requirements.txt

```diff
@@ -5,6 +5,7 @@ addict
 matplotlib
 addict
 numpy
+shutils
```

Comment:
this is not `shutil`. `shutil` is a built-in python library, so no need to install with `pip`.